packages/toolpad-components/src/Tabs.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { Tabs as MUITabs, Tab } from '@mui/material';
+import { createComponent } from '@mui/toolpad-core';
+
+interface TabProps {
+  title: string;
+  name: string;
+}
+
+interface Props {
+  active: string;
+  onChange: (value: number) => void;
+  tabs: TabProps[];
+  defaultValue: string;
+}
+
+function Tabs({ active, onChange, tabs, defaultValue }: Props) {
+  return (
+    <MUITabs
+      value={active || defaultValue}
+      onChange={(event, value) => {
+        onChange(value);
+      }}
+    >
+      {tabs.map(({ title, name }) => (
+        <Tab label={title} value={name} key={name} />
+      ))}
+    </MUITabs>
+  );
+}
+
+export default createComponent(Tabs, {
+  layoutDirection: 'horizontal',
+  argTypes: {
+    active: {
+      typeDef: { type: 'string' },
+      onChangeProp: 'onChange',
+      defaultValueProp: 'defaultValue',
+      helperText: 'Currently active tab.',
+    },
+    defaultValue: {
+      label: 'Default active tab',
+      typeDef: { type: 'string' },
+      defaultValue: 'tab-one',
+      helperText: 'The tab which will be active by default.',
+    },
+    tabs: {
+      typeDef: { type: 'array' },
+      defaultValue: [
+        {
+          title: 'Tab one',
```

Comment:
Why not use numbers instead of spelling them out? `Tab 1`, `Tab 2`, `Tab 3`?

Reply:
No reason, just what seemed to make tab title longer, anyways this is just for default values, so nothing prevents users to use whatever they want